crates/holochain/src/core/ribosome/host_fn/create.rs

```diff
@@ -298,7 +298,7 @@ pub mod wasm_test {
 
                 assert_eq!(record.unwrap().action_address(), &action_hash);
             },
-            4_000_000
+            11_000_000
```

Comment:
This seems pretty extreme. The default is 2MiB - we'd already set it to ~4Gb and now nearly tripling that? 

Does this actually get allocated? Should we be opening an issue to look at what's requiring this much stack memory?

Reply:
I think it's 4MB to 11MB, not GB https://docs.rs/tokio/latest/tokio/runtime/struct.Builder.html#method.thread_stack_size

Reply:
Ahh it is in bytes, fine, I can live with that :)